### PR TITLE
Boos 269 login error handling

### PR DESCRIPTION
**Pull request by anibalgatica**
Login shows a text message when login with wrong credentials and a toast when is a different kind of error

### CODE REVIEW

File: backend/Dockerfile [hunk 1]
```diff
@@ -7,4 +7,4 @@ RUN pip install --no-cache-dir -r requirements.txt
 
 COPY . .
 
-CMD [ "flask", "run", "--host=0.0.0.0"]
+CMD [ "flask", "run", "--host=0.0.0.0", "--port=5001"]
```
Comment (joaopcartier) [inline backend/Dockerfile:10]: Please let's try to not be committing port changes

File: docker-compose.yml [hunk 2]
```diff
@@ -18,5 +18,5 @@ services:
     volumes:
       - ./backend:/usr/src/app
     ports:
-      - 5000:5000
+      - 5001:5001
```
Comment (joaopcartier) [inline docker-compose.yml:21]: Please let's try to not be committing port changes

File: backend/Dockerfile (hunk 1 above)
Comment (joaopcartier) [inline backend/Dockerfile:10]: Please let's try to not be committing port changes

File: docker-compose.yml (hunk 2 above)
Comment (joaopcartier) [inline docker-compose.yml:21]: Please let's try to not be committing port changes

**Review by joaopcartier — APPROVED** [review]

File: backend/Dockerfile (hunk 1 above)
Comment (joaopcartier) [inline backend/Dockerfile:10]: Please let's try to not be committing port changes

File: docker-compose.yml (hunk 2 above)
Comment (joaopcartier) [inline docker-compose.yml:21]: Please let's try to not be committing port changes

**Review by joaopcartier — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (anibalgatica) [reply docker-compose.yml]: Changes reversed

**Pull request merged**